packages/jest-haste-map/src/lib/FSEventsWatcher.ts

```diff
@@ -11,7 +11,6 @@ import path from 'path';
 import {EventEmitter} from 'events';
 import anymatch from 'anymatch';
 import micromatch from 'micromatch';
-// eslint-disable-next-line
```

Comment:
This broke CI lint, undo?

Reply:
Interesting. I had to make this change in order to `yarn test` locally (it runs eslint with `--report-unused-disable-directives`). Removing for now.